### PR TITLE
db settings made, django settings file modified for beginning

### PR DESCRIPTION
**Pull request by Gormezoglu**

### CODE REVIEW

File: stracker/stracker/settings.py [hunk 1]
```diff
@@ -103,9 +108,21 @@
...
+def gettext_noop(s):
+    return s
+
+
+LANGUAGES = [
+    ("en", gettext_noop("English")),
+    ("tr", gettext_noop("Turkish")),
+    ("de", gettext_noop("German")),
+    ("fr", gettext_noop("French")),
+]
```
Comment (ttoygar) [inline stracker/stracker/settings.py:125]: I think we should start a poll about which languages will we add. 
 
 In any case 
 LOCALE_PATHS = (str(BASE_DIR / "location_of_localization_directory/"),)
 should be added.

**Review by ttoygar — CHANGES_REQUESTED** [review]
Great work. Thanks.

Can you update readme.md according to these changes too?

It should contain:
- [ ] Content of .env file
- [ ] Location of the example .env file
- [ ] environ package dependency in requirements.txt
- [ ] Database dependencies
- [ ] (If they are going to be added) Information about localization.

Great work. Thanks a lot.

**Comment by Gormezoglu** [comment]
> Great work. Thanks.
> 
> Can you update readme.md according to these changes too?
> 
> It should contain:
> 
> * [ ]  Content of .env file
> * [ ]  Location of the example .env file
> * [ ]  environ package dependency in requirements.txt
> * [ ]  Database dependencies
> * [ ]  (If they are going to be added) Information about localization.
> 
> Great work. Thanks a lot.

Sure, I will continue in the afternoon. 
Thx for the comments

File: stracker/stracker/settings.py (hunk 1 above)
Comment (smlblr) [reply stracker/stracker/settings.py:125]: I think it should be as little as possible. It can be TR and EN.

**Pull request closed without merging**